lib/galaxy/webapps/galaxy/api/jobs.py

```diff
@@ -181,7 +181,7 @@ def common_problems(self, trans, id, **kwd):
         has_duplicate_inputs = False
         for job_input_assoc in job.input_datasets:
             input_dataset_instance = job_input_assoc.dataset
-            if input_dataset_instance is None:
+            if not input_dataset_instance:
```

Comment:
Would that make a difference for the reported traceback? It says `'NoneType' object has no attribute 'get_total_size'` , so `input_dataset_instance` must be `None`.

Reply:
Thanks for pointing out https://github.com/galaxyproject/galaxy/pull/8194, I had concluded because the check is already `is None` that we'd have something like `''`. So I guess this is probably already fixed.

Reply:
ah, and of course that doesn't make sense, because `''` is not NoneType.